### DISCUSSION TITLE
Consent von FHIR Server

### DISCUSSION THREAD

**Discussion by trobanga** [medizininformatik-initiative/fts-next#431] [Ideas]
Ein FHIR Server anstelle von gICS

**Comment by SebStaeubert** [comment]
Dies ist eine Anforderung, die vom UK Bonn am 28.11.2024 in der PM-Telko gemeldet wurde.

**Comment by SebStaeubert** [comment]
Infos:
- [KDS Modul Consent](https://www.medizininformatik-initiative.de/Kerndatensatz/KDS_Consent_V2025/MII-IG-Modul-Consent-TechnischeImplementierung-FHIRProfile-Consent.html)
- Siehe [ClinFHIR-Darstellung](http://gb2.clinfhir.com#id1685953372265) (Graph Builder 2)
![image](https://github.com/user-attachments/assets/97ce6147-a16a-47b4-87bf-49be7ae027f5)

**Comment by knoppiks** [comment]
Nach Besprechung am 8.4.24 warten wir auf Feedback von @Tim-Steinbach-UKB zur Frage, ob Consent in Form von FHIR in der CD oder im Trust Center gemeint war.

**Comment by knoppiks** [comment]
Nach feedback, geht es um Consent im CD HDS

**Comment by trobanga** [comment]
Implementiert in 5.3
